src/App/DataSource/Email/Email.php

```diff
@@ -111,6 +113,15 @@ public function getOptions()
 		);
 	}
 
+	public function getInboundFields()
```

Comment:
This implementation of inbound field mappings drops the ability to map From and To values to post fields.
It was going to add a bit more mess to the code, but also I'm not convinced its a great idea to be able to map PII into the post fields.

@willdoran will losing this break anything that was using in COMRADES?